### PR TITLE
MM-47242 - improve tasklist items texts

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
Improve the tasklists texts. POC

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-47242

#### Related Pull Requests
n/a

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by pvev** [comment]
@marianunez @dontoisme @annelieseklein (please, ignore the repeated icons, I will update this along with the width size decisition)

With width of 500px:
Captura de Pantalla 2022-10-13 a las 9 37 24


With current width:
Captura de Pantalla 2022-10-13 a las 9 37 46

**Comment by annelieseklein** [comment]
@pvev 
please keep at current width - the text for 1st, 2nd and 3rd points will be: "Learn about messaging", "plan a sprint with kanban-style boards", "Resolve incidents faster with playbooks" ; keep the other points all the same text as current

**Review by annelieseklein — APPROVED** [review]

**Comment by pvev** [comment]
> @pvev please keep at current width - the text for 1st, 2nd and 3rd points will be: "Learn about messaging", "plan a sprint with kanban-style boards", "Resolve incidents faster with playbooks" ; keep the other points all the same text as current

ok, thanks for the feedback @annelieseklein, what about the icons?
@dontoisme ^^

**Comment by annelieseklein** [comment]
@pvev keep the icons for all items please, thank you! :)

File: components/onboarding_tasks/onboarding_tasks_manager.tsx [hunk 1]
```diff
@@ -44,55 +44,55 @@ const getCategory = makeGetCategory();
...
+            id='onboardingTask.checklist.task_resolve_incidents_faster_with_playbooks'
+            defaultMessage='🐛 Resolve incidents faster with playbooks'
         />
     ),
     [OnboardingTasksName.INVITE_PEOPLE]: (
         <FormattedMessage
-            id='onboardingTask.checklist.task_invite'
-            defaultMessage='Invite team members to the workspace'
+            id='onboardingTask.checklist.task_invite_people_to_team'
+            defaultMessage='👋 Enhance your Mattermost experience. Invite your team to check out our developer platform'
```
Comment (marianunez) [inline components/onboarding_tasks/onboarding_tasks_manager.tsx:65]: @pvev heads up that we only want to change the first top three items, everything from this one down should only add the emoji and no text change cc / @annelieseklein

**Review by marianunez — CHANGES_REQUESTED** [review]

**Review by marianunez — CHANGES_REQUESTED** [review]
@pvev changes LGTM but I still the text changes in the last items in the list as I mentioned here: https://github.com/mattermost/mattermost-webapp/pull/11335/files#r1001220301

**Review by marianunez — APPROVED** [review]
Thanks @pvev!

**Comment by marianunez** [comment]
@amyblais heads up we would like to cherry pick this change to `cloud`. It's a same text change for no major risks

**Review by JulienTant — APPROVED** [review]
LGTM ~ bit: I wonder if we should have "plain" emojis in translation files

**Review by julmondragon — APPROVED** [review]
LGTM!
nit: Would be good to update the description in ticket to reflect the final emoji/wordings implemented in this PR.

**Pull request merged**